.eslintrc.js

```diff
@@ -353,7 +353,7 @@ module.exports = {
     },
     {
       files: ['packages/*/src/**/*{.ts,.tsx,.js}'],
-      excludedFiles: ['*.d.ts', '*.spec.ts', '*.spec.tsx'],
+      excludedFiles: ['*.d.ts', '*.spec.ts', '*.spec.tsx', 'packages/mui-joy/**'],
```

Comment:
Why did you exclude this path?

Reply:
Otherwise I got an error if I use
```js
const props = useThemeProps({ name: 'JoyButton' }) // eslint error, the name should be `MuiButton`
```

Reply:
Let's fix the rule then (could be in a separate PR)